CHANGELOG

```diff
@@ -12,6 +12,8 @@ Version 4.3.1 (development)
 ===========================
 - Added support for hr-adaptivity using TMOP-based error estimator.
 
+- Updated hypre interface according to changes in version 2.22.1 of
```

Comment:
@tzanio, let me know if this description is fine. Thanks!

Reply:
looks good, thanks